### PR TITLE
#167727740 Allow users to post and get comments on a request

### PR DESCRIPTION
**Pull request by komsic**
#### What does this PR do?
This PR will allow users to post and get comments on a request

#### Description of Task to be completed?
- add comment migration and model files
- add comment seeder
- create endpoints to get and post comments for a specific request
- add tests

#### How should this be manually tested?
- CLONE this repo
- open the cloned directory
- run `npm run db:migrate`
- run `npm run seed`
- run `npm start:dev`
- access the docs using `{base-url}/docs`
- post a comment on a request using `{base-url}comments/{requestId}`
- get all the comments of a specific request using`{base-url}comments/{requestId}`

#### What are the relevant pivotal tracker stories?
[#167727740](https://www.pivotaltracker.com/story/show/167727740)

### CODE REVIEW

File: src/test/routes/comment.spec.js [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+        .set('authorization', managerToken)
+        .send(invalidContent)
+        .end((err, res) => {
+          expect(res.status).to.equal(400);
+          expect(res.body).to.have.property('status').that.equal('error');
+          done(err);
+        });
+    });
+
+    it('should return internal server error', (done) => {
```
Comment (ladaposamuel) [inline src/test/routes/comment.spec.js]: why should it return `internal server error`
I think the tests should be more descriptive.

**Review by ladaposamuel — COMMENTED** [review]
Good job

File: src/test/routes/comment.spec.js (hunk 1 above)
Comment (komsic) [reply src/test/routes/comment.spec.js]: Thank you for the feedback, I will make them more descriptive.

Comment (komsic) [reply src/test/routes/comment.spec.js]: Done. Please resolve the conversation

**Review by tejiri4 — APPROVED** [review]
LGTM

**Pull request merged**